README.md

```diff
@@ -1,7 +1,7 @@
-<p align="center"><img src="https://raw.githubusercontent.com/FormidableLabs/spectacle/main/docs/src/assets/logo_spectacle.png" width=250></p>
+<p align="center"><img src="https://raw.githubusercontent.com/FormidableLabs/spectacle/main/website/static/img/logo_spectacle.png" width=250></p>
 <h2 align="center">Spectacle Visual Editor</h2>
 
-Spectacle Visual Editor is a platform for building Spectacle decks visually using built-in Spectacle elements and templates created by the Formidable design team. The first phase of the project is building the visual editor using React, [Redux Toolkit](https://redux-toolkit.js.org) and [Segment Evergreen](https://evergreen.segment.com/components/). The second phase of the project will be adding a server-platform for users to store, host, and generate one-page outputs of their decks. A goal is also for users to be able to collaborate on a single link and make updates.
+Spectacle Visual Editor is an experimental platform for building Spectacle decks visually using built-in Spectacle elements and templates created by the Formidable design team. The first phase of the project is building the visual editor using React, [Redux Toolkit](https://redux-toolkit.js.org) and [Segment Evergreen](https://evergreen.segment.com/components/). The second phase of the project will be adding a server-platform for users to store, host, and generate one-page outputs of their decks. A goal is also for users to be able to collaborate on a single link and make updates.
```

Comment:
The link to Segment Evergreen throws a 404 error, looks like we need to get rid of the trailing slash. 
https://evergreen.segment.com/components